tests/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 from pytest_mock_resources.compat import sqlalchemy
 
+is_at_least_sqlalchemy2 = sqlalchemy.version.startswith("2.")
 is_sqlalchemy2 = sqlalchemy.version.startswith("1.4") or sqlalchemy.version.startswith("2.")
```

Comment:
if it starts with 1.4 then it is not 2.x - is this a misnomer?

Reply:
it definintely is. It was originally meant to deal with `.ext.declarative` vs `.orm`, which a sqlalchemy2-style thing. oh well

Reply:
gotcha